kikuchipy/signals/ebsd_master_pattern.py

```diff
@@ -61,6 +74,145 @@ def __init__(self, *args, **kwargs):
         self.projection = kwargs.pop("projection", None)
         self.hemisphere = kwargs.pop("hemisphere", None)
 
+    def get_patterns(
+        self,
+        rotations: Rotation,
+        detector: EBSDDetector,
+        energy: int,
+        n_chunk: Optional[int] = -1,
+        dtype_out: Optional[np.dtype] = np.float32,
+        compute: Optional[bool] = False,
+    ) -> Union[EBSD, LazyEBSD]:
+        """Creates a dictionary of EBSD patterns for a sample in the
+        (RD, TD, ND) reference frame, given a set of local crystal lattice
+        rotations and a detector model from a master pattern in the Lambert
+        projection.
+
+        Parameters
+        ----------
+        rotations : Rotation
+            Set of unit cell rotations to get patterns from.
+        detector : EBSDDetector
+            EBSDDetector object describing the detector geometry with one
```

Comment:
I suggest rephrasing to "with a single, fixed projection/pattern center", since pattern center is most used in the literature.